ql/src/test/org/apache/hadoop/hive/ql/metadata/TestHiveMetaStoreChecker.java

```diff
@@ -330,17 +330,6 @@ public void testPartitionsCheck() throws HiveException,
     assertEquals(partToRemove.getTable().getTableName(),
         result.getPartitionsNotOnFs().iterator().next().getTableName());
     assertEquals(Collections.<CheckResult.PartitionResult>emptySet(), result.getPartitionsNotInMs());
-
-    List<Map<String, String>> partsCopy = new ArrayList<Map<String, String>>();
-    partsCopy.add(partitions.get(1).getSpec());
```

Comment:
is there a successor of this test?

Reply:
I think the qtests handles most of cases